### DISCUSSION TITLE
admin.site.site_header = "KD Connector Admin" not working

### DISCUSSION THREAD

**Discussion by mikeoliversinag** [fabiocaccamo/django-admin-interface#298] [Q&A]
I added admin.site.site_header = "KD Connector Admin" to my home site urls.py but it did not take effect in the Admin pages  Still showing 'Django administration'

I was able to add it in the theme however.  its ok but I wonder why.

**Comment by fabiocaccamo** [comment]
There is an old issue about the `admin.site.site_header` #75

Summary: the theme title value overrides the `admin.site.site_header` value, so if you want to set the `admin.site.site_header` programmatically, ensure that the theme title is unset (empty string) to let it fallback to the admin site option.
